<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1652</width>
    <height>916</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Подключение COM-порта</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>20</y>
       <width>275</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="comL"/>
      </item>
      <item>
       <widget class="QPushButton" name="BOpen">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BClose">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>501</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Аккумуляторы</string>
    </property>
    <widget class="QProgressBar" name="OsVolB">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>31</width>
       <height>101</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>75</number>
     </property>
     <property name="maximum">
      <number>130</number>
     </property>
     <property name="value">
      <number>75</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QProgressBar" name="PSVolB">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>31</width>
       <height>101</height>
      </rect>
     </property>
     <property name="maximum">
      <number>12</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Основной</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>140</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Дочерний</string>
     </property>
    </widget>
    <widget class="QLabel" name="OsVolL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="PSVolL">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="PSTL">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>140</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Температура</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="PSTB">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>31</width>
       <height>101</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-10</number>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="OsTV">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Температура</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="OsTB">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>31</width>
       <height>101</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-10</number>
     </property>
     <property name="maximum">
      <number>60</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="OsHL">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Состояние нагревателя</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="Signal">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>signal</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>10</y>
      <width>591</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Мгновенное значение радиации</string>
    </property>
    <widget class="PlotWidget" name="GraphRD" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>571</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="OsRD">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>270</y>
      <width>601</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Статическое значение радиации</string>
    </property>
    <widget class="PlotWidget" name="GraphRS" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>571</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="OsRS">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>411</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Высота на пикоспутнике</string>
    </property>
    <widget class="PlotWidget" name="GraphHP" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>381</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="OsHP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_10">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>520</y>
      <width>421</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>Давление на пикоспутнике</string>
    </property>
    <widget class="PlotWidget" name="GraphPP" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>391</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="OsPP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_11">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>530</y>
      <width>401</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Температура пикоспутника</string>
    </property>
    <widget class="PlotWidget" name="GraphTP" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>381</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="OsTOutP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>461</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Температура на плате</string>
    </property>
    <widget class="PlotWidget" name="GraphT" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>421</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="OsTOut">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>461</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Акселерометр</string>
    </property>
    <widget class="PlotWidget" name="GraphA" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>421</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="OsA">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1652</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
